ariadne/cardano/src/Ariadne/Wallet/Cardano/Kernel/Keystore.hs

```diff
@@ -251,8 +206,8 @@ delete walletId (Keystore ks) = do
 -- | Returns all the 'EncryptedSecretKey' known to this 'Keystore'.
 toList :: Keystore -> IO [(WalletId, EncryptedSecretKey)]
 toList (Keystore ks) =
-    withMVar ks $ \(InternalStorage us) ->
-        pure $ map (first hashPubKeyToWalletId) $ toPairs $ us ^. usWallets
+    withMVar ks $ \us ->
+        pure $ map (first hashPubKeyToWalletId) $ Map.toList $ us ^. usWallets
```

Comment:
Why replace `toPairs` here?

Reply:
I didn't realize then that they are essentially same thing. Will revert.